src/utils/index.ts

```diff
@@ -1,8 +1,9 @@
-export { CombinedError } from './error';
-export { getKeyForRequest } from './keyForQuery';
-export { createRequest } from './request';
-export { formatDocument, collectTypesFromResponse } from './typenames';
-export { toSuspenseSource } from './toSuspenseSource';
+export * from './debug';
```

Comment:
It'd be nice to revert this tbh 😅

It's less convenient but this way we just have a list here of what's supposed to be exposed. Fine to merge though

Reply:
I wasn't expecting the utils index file to look like it did as the explicit re-exporting feels pretty confusing. If we're adopting the standard of importing utilities from the 'utils' folder, what advantage do we get by re-stating the exported values? Feels like more work! 😕

I've added explicit exports at the root level index file so we're not exposing internal details to end users 🔊

Reply:
The main reason for the change here was because we're filtering internal usage of exports in our utils folder but then not filtering when exporting to the end user - I've flipped that the other way round which makes more sense imho